packages/react-devtools-shared/src/hooks/parseHookNames/parseSourceAndMetadata.js

```diff
@@ -635,32 +617,40 @@ function parseSourceMaps(
       }
 
       const sourceMapJSON = hookSourceAndMetadata.sourceMapJSON;
-      if (sourceMapJSON != null) {
-        hookParsedMetadata.metadataConsumer = withSyncPerfMeasurements(
-          'new SourceMapMetadataConsumer(sourceMapJSON)',
-          () => new SourceMapMetadataConsumer(sourceMapJSON),
-        );
-        hookParsedMetadata.sourceConsumer = withSyncPerfMeasurements(
-          'new SourceMapConsumer(sourceMapJSON)',
-          () => new SourceMapConsumer(sourceMapJSON),
-        );
 
-        const runtimeSourceURL = hookSourceAndMetadata.runtimeSourceURL;
+      if (hookParsedMetadata.sourceConsumer === null) {
```

Comment:
nit: should we do a soft null check here in case it's undefined? (i.e. sourceConsumer == null)

Reply:
i guess it doesn't really matter, but for this branch `parseSourceMaps` shouldn't we also be checking the `runtimeURLToMetadataCache`?

Reply:
This branch was deleted before you commented too :)